getting-started/trino-config/catalog/iceberg.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2024 Snowflake Computing Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+connector.name=iceberg
+iceberg.catalog.type=rest
+iceberg.rest-catalog.uri=http://polaris:8181/api/catalog
+iceberg.rest-catalog.security=OAUTH2
```

Comment:
Should it also include `iceberg.rest-catalog.oauth2.scope`? It seems, non scoped requests fail with 
```
org.apache.iceberg.exceptions.ForbiddenException: Forbidden: Principal 'XXX' with activated PrincipalRoles '[catalog]' and activated ids '[]' is not authorized for op XXX
```
Related trino PR - https://github.com/trinodb/trino/pull/22961

Reply:
I don't think https://github.com/trinodb/trino/pull/22961 is required in case of `iceberg.rest-catalog.oauth2.token`. The scope is needed for `iceberg.rest-catalog.oauth2.credential` in my understanding.